src/docs/version-conf.js

```diff
@@ -0,0 +1 @@
+conf = { version: '7.6.0-test'}
```

Comment:
It would be nice to reuse the _dev_ suffix instead of _test_ since it is already used to locate the last documentation in the URL:

http://doc.activeeon.com/dev/